pkg/components/gangway/component.go

```diff
@@ -308,6 +308,7 @@ func (c *component) LoadConfig(configBody *hcl.Body, evalContext *hcl.EvalContex
 	return gohcl.DecodeBody(*configBody, evalContext, c)
 }
 
+// TODO: Convert to helm chart.
```

Comment:
```suggestion
// TODO: Convert to Helm chart.
```

If you change this, please fix the rest of the occurrences as well.